ckanext/custom_schema/schemas/dataset.yaml

```diff
@@ -44,20 +44,39 @@ dataset_fields:
   help_text: License definitions and additional information can be found at http://opendefinition.org/
 
 ## "department"
-#  The Department field will be mapped to CKAN organizations
+#  The Group field will be mapped to CKAN organizations
 - field_name: owner_org
-  label: Department
+  label: Group
```

Comment:
Labeling Organizations as Groups can be confusing to users since CKAN documentation can refer to groups as something different.